source/common/http/http3_status_tracker.cc

```diff
@@ -0,0 +1,46 @@
+#include "common/http/http3_status_tracker.h"
+
+namespace Envoy {
+namespace Http {
+
+namespace {
+
+// Initially, HTTP/3 is be marked broken for 5 minutes.
+const std::chrono::minutes DefaultExpirationTime{5};
```

Comment:
nit: https://github.com/envoyproxy/envoy/blob/main/STYLE.md#deviations-from-google-c-style-guidelines has a constant naming rule that's a bit different from Google's.
So in the Envoy world, I guess this should be named ConstantDefaultExpirationTime.

Reply:
Interesting! I *think* that style guide is somewhat confusing. Here's what it says:

> The Google C++ style guide points out that constant vars should be named kConstantVar. In the Envoy codebase we use ConstantVar or CONSTANT_VAR. 

So both of the Envoy names do start with "constant". But so does the Google name "kConstantVar" which points to an example variable named kDaysInAWeek. I think the point is just that constants need to be ALL_UPPER_CASE or need to be AllMixedCases, for example DaysInAWeek or DAYS_IN_A_WEEK.

Looking at the code, I think that matches the convention used there. WDYT?

Reply:
Oh great observations!
I thought "Constant" was a required prefix. But looking at the codebase, I think "Constant" here is the same as "Foo".